src/main/java/seedu/address/logic/commands/ListAppointmentsCommand.java

```diff
@@ -77,8 +78,14 @@ private String formatAppointments(List<Person> appointments) {
 
         return appointments.stream()
                 .map(person -> {
-                    LocalDateTime dateTime = LocalDateTime.parse(person.getSchedule().dateTime, inputFormatter);
-                    return String.format("%s: %s", person.getName(), dateTime.format(outputFormatter));
+                    Name personName = person.getName();
```

Comment:
Is this code a bit nested? Is it possible to break the nested stream operations into smaller methods?